### DISCUSSION TITLE
Annoying problems - cannot finish tutorial, no help link, blank screens

### DISCUSSION THREAD

**Discussion by adaptivebehaviour** [Budibase/budibase#9899] [Help]
Hi, I've just started using Budibase for the first time this morning. So far the idea looks fantastic and I can certainly see the potential for using this in the future, but there are some really annoying problems that are discouraging me from using it any more:

 - The final step of the welcome tutorial shows EVERY single time I do anything - click on a component, change screens e.g. from Data to Design - anything at all - the Publish 4/4 screen shows. Because it obscures the tools, I have to click on "Finish" every time I do anything.
 
![image](https://user-images.githubusercontent.com/110525929/223241338-35e987b0-b107-46b2-997c-b041a869a61c.png)

 - There's no help/documentation link on the top navigation bar. I find this quite surprising as it's a staple of SaaS. For example, Supabase has this at the top of every screen:
 
![image](https://user-images.githubusercontent.com/110525929/223241881-4b40e644-6f32-4257-a96e-36d3ff99524a.png)

  - Every time I click on a button that results in loading a new screen nothing happens - for ages. There's no feedback from clicking the button, so I don't know whether or not the action has registered, and then when the change is finally initiated, the screen goes completely blank and then stays that way for ages, making it look like the application has hung. 
  
It would be great if someone could let me know how I can stop the tutorial step from showing every time - the other things are annoying, but this tutorial thing is preventing me from using the app.

**Comment by shogunpurple** [comment]
@adaptivebehaviour are you still seeing this behaviour in cloud? There was a small blip there that might have affected it. If it's still happening I'll help debug.

**Reply by adaptivebehaviour** [reply]
Yep, still got it. I just logged back in to check and the tutorial started again:

![image](https://user-images.githubusercontent.com/110525929/223271063-aaae3438-ef21-4bb4-9353-1a57bd7ce0d1.png)

I went though it all and clicked "Finish" at the end but the final popup still keeps showing every time I click on something.

A notification for an update popped up so I updated, logged back out and back in again, but it made no difference. Tutorial again, final screen pops up every time I click on something etc:

![image](https://user-images.githubusercontent.com/110525929/223273869-3db030c6-5558-4f67-ace1-f5d3d19a87c9.png)

There are 5 steps now - the user step was added in the update I think.

**Reply by shogunpurple** [reply]
Ok, sorry to hear that! What is your tenant ID, and I'll see if I can fix on our end

**Reply by adaptivebehaviour** [reply]
I can't find any reference to a tenant ID anywhere - are you referring to "adaptivebehaviour"? If not, where can I find the tenant ID?

**Reply by shogunpurple** [reply]
It's in your URL in cloud - `<tenant>.budibase.app`

**Reply by adaptivebehaviour** [reply]
Yes that's "adaptivebehaviour"

**Comment by PClmnt** [comment]
@adaptivebehaviour Are there any errors in the browser console whenever you click "Finish" on the tour?

**Reply by adaptivebehaviour** [reply]
I just logged in and got the tour again. However, when I initially went into the project I got this:

![image](https://user-images.githubusercontent.com/110525929/223535279-ca8805a2-e1dd-422e-b13a-57d2f409664b.png)

...and then I got this when I clicked on "Finish", and on every subsequent click:

![image](https://user-images.githubusercontent.com/110525929/223535517-e22ff54c-010e-4cae-a81e-c2e4e6e7ee2c.png)

**Reply by adaptivebehaviour** [reply]
Note that I am logging in through my Google account - so no username or password is provided directly to Budibase.

**Reply by shogunpurple** [reply]
This should now be resolved!

**Reply by adaptivebehaviour** [reply]
Yes it is - thank you!
Are there any plans to have a "busy" animation running while pages are loading? It would really help in knowing whether or not the page is actually loading or if it has hung.
It would also be really good to have a link to documentation/support somewhere on the Data/Design/Automate pages - unless there is one and I can't see it?

**Reply by shogunpurple** [reply]
@adaptivebehaviour - I'm not sure what you mean. We have skeleton loaders and spinners that show active loading states for apps and in the builder.

Documentation/support links can be found in the portal at the bottom right:
<img width="372" alt="Screenshot 2023-03-07 at 23 40 37" src="https://user-images.githubusercontent.com/11256663/223580587-18c070f7-8b6b-4424-945d-633598275292.png">
